Improve filesystem check, improve wording

easyaspi314 opened the pull request:
I made it so that the script checks that the data partition is either F2FS or EXT4, not if it is not vFAT. 

However, I tried to improve it to make more sense by giving more, informal, information, and I directed people to post 2. 

Because it will hopefully further prevent [this](http://forum.xda-developers.com/showpost.php?p=68981820&postcount=1093). -_-

easyaspi314 committed:
Improve filesystem check, improve wording

I made it so that the script checks that the data partition is either F2FS or EXT4, not if it is not vFAT. 

However, I tried to improve it to make more sense, and direct people to post 2. 

Because it will hopefully further prevent [this](http://forum.xda-developers.com/showpost.php?p=68981820&postcount=1093). -_-

thewizrd commented:
I'm ok with the data format changes, but the use of "my" and "I" seem off to me. Could you change the wording a bit?

easyaspi314 committed:
Remove "I", "My", and simplify `if` statements.

I really want to do `&&`, being a Java programmer (and learning C++ at the moment). 

But that is a Bashism. :(

easyaspi314 commented:
"My" and "I" are removed.

I also simplified the if statements using `-o`. It is really tempting to do `&&`. 

Damn Bashisms. Breaking POSIX to appeal to Java/C programmers.
